lua/fennel_select/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                fennel_select
+version             0.1
+revision            0
+categories          lua
+platforms           darwin
+license             MIT
```

Comment:
this should probably the same as for all other "select" Portfiles and MacPorts --> BSD

Reply:
I copied this from the `luacheck_select` Portfile, should I update that one in this PR as well?